pytrip/lib/core.c

```diff
@@ -641,12 +644,62 @@ static PyObject * calculate_lvh_slice(PyObject *self, PyObject *args)
     return PyArray_Return(vec_out);
 }
 
+static void append_intersection(PyObject *list_out, double x, double y, double z){
```

Comment:
I think this should be rather called `append_point_to_list`
The method is not specific to intersection